src/include/miopen/solver.hpp

```diff
@@ -2400,8 +2494,15 @@ struct ConvOclBwdWrW2 : ConvTunableSolver<PerformanceConfigConvOclBwdWrw2<N_BATC
 #if defined(__clang__) && defined(CONV_OCL_DIR2D_BWDWRW_2_CPP)
 #pragma clang diagnostic push
 #pragma clang diagnostic ignored "-Wweak-template-vtables"
+#pragma clang diagnostic ignored "-Wunneeded-internal-declaration"
```

Comment:
Why do you need to suppress this warning?

Reply:
removed